### PR TITLE
documentation: create cluster.article

### PR DESCRIPTION
**Pull request by floren**
installing.article no longer contains information about setting up the network. Instead,
the new file cluster.article describes in more detail how you can set up a cluster
to run minimega.

### CODE REVIEW

**Comment by floren** [comment]
@jcrussell @djfritz

File: doc/content/articles/installing.article [hunk 1]
```diff
@@ -5,40 +5,52 @@ The minimega authors
...
 
-First, obtain the minimega source, then run the all.bash script from the top of
-the repo:
+Next, check out the 2.3 release. If you wish to run the development
+version ("tip") of minimega, skip this command.
 
-       git clone git@github.com:sandia-minimega/minimega.git
-       cd minimega
-       ./all.bash
+	$ git checkout tags/2.3
```
Comment (jcrussell) [inline doc/content/articles/installing.article]: Should be `git checkout 2.3`.

File: doc/content/articles/installing.article [hunk 2]
```diff
@@ -55,36 +67,38 @@ minimega.
...
-in `$PATH`. If you plan to use minimega on a head node and only send messages
-to other nodes to launch and control VMs, you can ignore any external tool
-warnings at launch. 
+minimega also has a number of external tools it executes. When you
+start minimega, it will check to see if each of the tools it may need
+are available in `$PATH`. Depending on your intended use case, you may
+not need every single external program.
 
-If you plan to launch and maintain VMs, you'll need the following programs:
+If you plan to launch and maintain VMs, you'll need the following programs at a minimum:
```
Comment (jcrussell) [inline doc/content/articles/installing.article]: Now need Open vSwitch v1.11+

File: doc/content/articles/installing.article [hunk 3]
```diff
@@ -55,36 +67,38 @@ minimega.
...
 
 - kvm - qemu-kvm with the kvm kernel module loaded (minimum version 1.6)
 - ip - ip tool for manipulating devices
 - ovs-vsctl - Open vSwitch switch control with daemon running and kernel module loaded (minimum version 1.4)
 - ovs-ofctl - Open vSwitch openflow control with daemon running and kernel module loaded
+
+We also recommend installing the following; they are not strictly
+necessary for basic VM use but are required for some more advanced
+operations:
+
```
Comment (jcrussell) [inline doc/content/articles/installing.article]: Dnsmasq v2.73+

File: doc/content/articles/cluster.article [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+	$ /opt/minimega/bin/minimega -attach
+
+This will give you a minimega prompt where we'll enter the command to
+launch on the other nodes:
+
+	minimega$ deploy launch node[2-10]
+
+The `deploy` command copies the current minimega binary to the nodes
+you specify using `scp`, then launches them with `ssh` using the same
+set of command line flags as the minimega instance that ran `deploy`.
```
Comment (jcrussell) [inline doc/content/articles/cluster.article]: Should maybe add a note that the deploy API assumes password-less login.

**Pull request merged**